### ISSUE TITLE
Add support for PostGIS data types

### ISSUE THREAD

**Issue by bchapuis** [bytefish/PgBulkInsert#35]
The Postgis extension comes with some additional data types, which are very popular within in the GIS community. 
https://postgis.net/docs/using_postgis_dbmanagement.html

It is unclear to me how such geometries can be inserted in the database using the copy manager (WKB format?), but it would be great to support them as well.

**Comment by bchapuis** [comment]
@bytefish apparently, the WKB format can be used with COPY, so I started working on a prototype version to support the POSTGIS extension. First things first, I tried to execute all the unit tests of PgBulkInsert locally and it seems that I have to setup a local database and a schema. I havn't found SQL initialization scripts in the repository and I was wondering if something else was needed to execute these tests locally. Thanks a lot for your help :smile:
